basket/news/backends/ctms.py

```diff
@@ -83,6 +83,7 @@ class CTMSError(Exception):
         "mofo_relevant": "mofo_relevant",
     },
     "vpn_waitlist": {"geo": "fpn_country", "platform": "fpn_platform"},
+    "relay_waitlist": {"geo": "relay_country"},
```

Comment:
Add this `relay_country` to the tests so that we can test we will receive it as `{"relay_waitlist": {"geo": "XX"}}`
[    "fpn_country": "fr",](https://github.com/mozmeao/basket/blob/8b6f57379875f9d8c94428e3cd6be2af9f3b9152/basket/news/tests/test_ctms.py#L105)